uwblueprint/sistering
# Create Rich Text Display Component

## Ticket link

Closes #130 



## Implementation description
* Created a rich text display component to be used in admin "review and "post"
<img width="1057" alt="Screen Shot 2022-03-19 at 11 42 08 PM" src="https://user-images.githubusercontent.com/82241706/159151352-9515931b-0619-4cdd-93e3-b89e35f513ed.png">





## Steps to test
1. Seed the database (run "docker exec -it sistering_backend bash -c "npx prisma db seed")
2. See richTextSamples in comments in CreatePostingReview.tsx
3. Set the child of the RichTextDisplay component used on line 71 of CreatePostingReview.tsx to the different richTextSamples
4. Navigate to http://localhost:3000/admin/posting/create/review to see the richText displayed 

Expected Results (what Albert sees when making the richText components): 

richTextSample1
![richtextsample1](https://user-images.githubusercontent.com/82241706/159179305-a6d10bf1-db15-43db-8a02-9e00bfe4be5a.png)

richTextSample2
![richtextsample2](https://user-images.githubusercontent.com/82241706/159179312-0f039692-519e-4b91-9f0a-451ed3417331.png)

richTextSample3
![richtextsample3](https://user-images.githubusercontent.com/82241706/159179324-06312db0-6b47-42c8-a3bf-1bd87254b994.png)

richTextSample4
<img width="781" alt="Screen Shot 2022-03-20 at 1 20 58 PM" src="https://user-images.githubusercontent.com/82241706/159184453-70938dc3-0e12-4302-b386-fe0c8cddb1b6.png">


## What should reviewers focus on?
* RichTextDisplay component works properly 


## Checklist
- [ x] My PR name is descriptive and in imperative tense
- [ x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ x] I have run the appropriate linter(s)
- [x ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
